.github/workflows/super-linter.yml

```diff
@@ -31,7 +31,7 @@ jobs:
           # that .golangci.yml is available to the golangci-lint linter
           # https://github.com/github/super-linter/blob/main/docs/using-rules-files.md
           LINTER_RULES_PATH: /
-          VALIDATE_CLANG_FORMAT: true
+          VALIDATE_CLANG_FORMAT: flase
```

Comment:
"false" is spelled incorrectly. But also, why not validate the clang format?

Reply:
I pushed a fix for the spelling error. The reason is appears the super-linter does not use the latest clang-format version or I could not find a way to have it use 15.0.2. The clang-format action does allow us to pick the correct version. I was trying to fix the problem in PR #250 as it works with clang-format version 15 on my machine.

I we can find a better solution and still use super-linter it would be great.